kernel/init.c

```diff
@@ -167,6 +167,10 @@ void _data_copy(void)
 {
 	(void)memcpy(&__data_ram_start, &__data_rom_start,
 		 ((u32_t) &__data_ram_end - (u32_t) &__data_ram_start));
+#ifdef CONFIG_RAM_FUNCTION
```

Comment:
A bit late on this, but i think we now have a convention of having only one underscore in the beginning of variables.
See _app_smem_start.
could you change this?